test/blackbox-tests/test-cases/cinaps.t/include-subdirs.t

```diff
@@ -20,6 +20,12 @@ cinaps doesn't work with (include_subdirs unqualified)
   > let x = 1
   > EOF
 
-  $ dune runtest --diff-command diff 2>&1 | head -2
-  Error: exception { exn =
-      ("Directory creates generated directories inside its descendant source
+  $ dune runtest --diff-command diff
+            sh (internal) (exit 1)
+  (cd _build/default && /bin/sh -c 'diff sub/test.ml sub/test.ml.cinaps-corrected')
```

Comment:
For me dune prints "/usr/bin/sh" instead of "/bin/sh". :-\
In fact cinaps.t/run.t has "/usr/bin/sh" too, so I'm a bit confused.

Is cinaps.t/run.t failing for you, then?

Reply:
In #3630 I've added `2>&1 | sed -E 's/[^ ]+sh/\$sh/'` to get around this problem. We should probably do that here too.

Reply:
Looks a bit ugly, but if it solves the problem then go for it.